### PR TITLE
ChangedGenotypeGVCfsDoc

### PR DESCRIPTION
**Pull request by chandrans**
Added some minor changes for easier reading. Should insert some docs for CombineGVCFs when we know where the doc lives.

### CODE REVIEW

**Comment by sooheelee** [comment]
CombineGVCFs hasn't been ported to GATK4 @chandrans. If people need to use it, they will have to use the GATK3 version. To replace it, there are GenomicsDB tools instead. 

In lieu of mentioning a new tool we haven't yet documented, perhaps we can mention that CombineGVCFs is only in GATK3?

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java [hunk 1]
```diff
@@ -26,35 +26,48 @@
...
+ *   -O output.vcf
+ * </pre>
+ *
+ * <h4>Perform joint genotyping on a set of GVCFs listed in a text file, one per line</h4>
  * <pre>
- * gatk-launch GenotypeGVCFs \
+ * ./gatk-launch GenotypeGVCFs \
  *   -R reference.fasta \
- *   -V sample1.g.vcf \
+ *   -V input_gvcfs.list \
```
Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java]: @droazen we can still do this in GATK4, right?

**Comment by vdauwera** [comment]
one question for @droazen or @lbergelson re: list input capabilities

**Comment by vdauwera** [comment]
@sooheelee we'll document that separately as part of the pipeline docs; that doesn't belong in individual tool docs imho

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java (hunk 1 above)
Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java]: (I'm going to assume this is the case)

**Comment by vdauwera** [comment]
will merge when tests pass

File: src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java (hunk 1 above)
Comment (ronlevine) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java]: Should this be camelCase, for consistency?

Comment (vdauwera) [inline src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java]: I prefer snake case for file names as it's easier to parse.

**Comment by lbergelson** [comment]
@vdauwera Any gatk4 argument that takes a collection can accept a list file.

**Pull request merged**